grants.yml

```diff
@@ -3061,3 +3061,111 @@
   to:
     - roles:
         - login-identity:mozilla-auth0/ad|Mozilla-LDAP|bchatterton
+
+### landoscript scopes
+# on try, we're fairly liberal with these scopes for ease of testing, and
+# allow all actions to be run through any means.
+- grant:
+    - project:releng:lando:action:*
+    # this dev/test repo will probably change; but it's what we have for now
+    - project:releng:lando:repo:ff-test-dev
+  to:
+    - project:
+        alias: try
+
+# on level 3 repositories we only allow the specific actions that we expect to
+# run in a given repository, and scope the grants down to the cron or action
+# that needs them, mostly to prevent accidents.
+
+# non-merge day actions are all quite straightforward: projects only need
+# a repo scope for themselves, and a specific action scope, on one or more jobs.
+- grant:
+    - project:releng:lando:action:android_l10n_import
+    - project:releng:lando:repo:{lando_repo}
+  to:
+    - project:
+        alias: autoland
+        job: cron:android-l10n-import
+
+- grant:
+    - project:releng:lando:action:android_l10n_sync
+    - project:releng:lando:repo:{lando_repo}
+  to:
+    - project:
+        alias: mozilla-beta
+        job: cron:android-l10n-sync
+
+- grant:
+    - project:releng:lando:action:l10n_bump
+    - project:releng:lando:repo:{lando_repo}
+  to:
+    - project:
+        alias: [autoland, mozilla-beta]
+        job: cron:l10n-bumper
+
+- grant:
+    - project:releng:lando:action:tag
+    - project:releng:lando:repo:{lando_repo}
+  to:
+    - project:
+        alias: [mozilla-beta, mozilla-release, mozilla-esr115, mozilla-esr128]
```

Comment:
Can we use a wildcard for the `mozilla-esr` projects? (I'm actually asking, I'm not sure)

Reply:
I don't know either, but I'm going to try it out!

Reply:
This doesn't work, unfortunately! It's a nice idea though, particularly for esr branches...